inference-engine/src/mkldnn_plugin/utils/shape_inference/shape_inference.cpp

```diff
@@ -24,10 +28,43 @@ void shape_inference(ov::Node* op,
         bool status = resolve_auto_pad_for_shape(node, pads_begin, pads_end, input_shapes, 2, 2);
         OPENVINO_ASSERT(status, "Convolution shape inference doesn't have enough information to calculate static shapes");
         shape_infer(node, pads_begin, pads_end, input_shapes, output_shapes);
+    } else if (auto node = ov::as_type<ov::opset8::GroupConvolution>(op)) {
+        ov::CoordinateDiff pads_begin, pads_end;
+        bool status = resolve_auto_pad_for_shape(node, pads_begin, pads_end, input_shapes, 2, 3);
+        OPENVINO_ASSERT(status, "GroupConvolution shape inference doesn't have enough information to calculate static shapes");
+        shape_infer(node, pads_begin, pads_end, input_shapes, output_shapes);
+    } else if (auto node = ov::as_type<ov::opset8::ConvolutionBackpropData>(op)) {
+        ov::CoordinateDiff pads_begin, pads_end;
+        ov::StaticShape output_shape_input;
+        get_data_as_shape<ov::StaticShape>(2, op, output_shape_input, constant_data);
```

Comment:
maybe I missed something, but as far as I understand 3rd input is optional for `ConvolutionBackpropData`. 
So when we call `const auto& constant = ov::as_type_ptr<ov::opset1::Constant>(op->get_input_node_shared_ptr(idx));` for idx == 2 we got exception in  `ConvolutionBackpropData` has only 2 inputs